README.md

```diff
@@ -19,11 +19,22 @@ If you need help using or installing Node.js, please use the
 
 ## Resources for Newcomers
 
+### Official Resources
+
 * [Website](https://nodejs.org/en/)
 * [Contributing to the project](./CONTRIBUTING.md)
-* IRC (general questions): [#node.js on chat.freenode.net](https://webchat.freenode.net?channels=node.js&uio=d4)
 * IRC (node core development): [#node-dev on chat.freenode.net](https://webchat.freenode.net?channels=node-dev&uio=d4)
 
+### Community Resources
```

Comment:
Using the term "Community" here to differentiate with official resources might make it sound like the rest of the resources are not part of the community. What do you think of rephrasing it as "Non official resources"?